src/dmd/identifier.d

```diff
@@ -125,18 +125,6 @@ nothrow:
 
     extern (C++) __gshared StringTable stringtable;
 
-    /**
-       A secondary string table is used to guarantee that we generate unique
-       identifiers per module. See generateIdWithLoc and issues
-       https://issues.dlang.org/show_bug.cgi?id=16995
-       https://issues.dlang.org/show_bug.cgi?id=18097
-       https://issues.dlang.org/show_bug.cgi?id=18111
-       https://issues.dlang.org/show_bug.cgi?id=18880
-       https://issues.dlang.org/show_bug.cgi?id=18868
-       https://issues.dlang.org/show_bug.cgi?id=19058.
-     */
-    private extern (C++) __gshared StringTable fullPathStringTable;
```

Comment:
This is the fifth time that someone has completely reimplemented this, right?

Reply:
Probably. But this is only an attempt to simplify/accelerate the current version, without functional changes.